include/opendht/http.h

```diff
@@ -244,6 +246,7 @@ class OPENDHT_PUBLIC Request
     void add_on_state_change_callback(OnStateChangeCb cb);
 
     void send();
+    void send(const Json::Value& json, OnStateChangeCb cb = {});
```

Comment:
Should have a wrapper callback that returns a json object and a potential error on done